pkg/api/util/group_version.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import "strings"
+
+func GetVersion(groupVersion string) string {
```

Comment:
Please define a GroupVersion type.

```
type GroupVersion struct {
  Group string
  Version string
}
```

Instead of these GetVersion/GetGroup functions, you should provide a `func ParseGroupVersion(gv string) (GroupVersion, err)`.

Also, I suggest not putting this in a util package. Maybe start a "pkg/api/groups" package?